packages/plugin/src/Integrations/MailingLists/Mailchimp/BaseMailchimpIntegration.php

```diff
@@ -23,19 +23,19 @@
 use Solspace\Freeform\Library\Integrations\Types\MailingLists\DataObjects\ListObject;
 use Solspace\Freeform\Library\Integrations\Types\MailingLists\MailingListIntegration;
 
-abstract class BaseMailChimpIntegration extends MailingListIntegration implements OAuth2ConnectorInterface, MailChimpIntegrationInterface
+abstract class BaseMailchimpIntegration extends MailingListIntegration implements OAuth2ConnectorInterface, MailchimpIntegrationInterface
 {
     use OAuth2Trait;
 
     protected const LOG_CATEGORY = 'Mailchimp';
 
-    protected const CATEGORY_MEMBERS = 'Members';
+    protected const CATEGORY_CONTACT = 'Contact';
```

Comment:
Yep, wasn't consistent. Few used plurals, some didn't. Then Kelsey updating the language to use contacts instead of members, as Mailchimp docs and dashboard refers to contacts and not members, so also changed for consistency.